crates/swc_ecma_transforms_compat/tests/fixture/async-to-generator/issue-2556/1/exec.js

```diff
@@ -0,0 +1,21 @@
+async function init() {
+    return fib;
+
+    const fib = 55;
```

Comment:
@kdy1 I add the fib verbal here which is unreachable code.
It should be removed in the transform process.
But the actual behavior does not match.

Reply:
Can you post the transformed code?
I think `resolver` has a bug related to scoping of `const` variables.